tests/integration/conftest.py

```diff
@@ -611,14 +611,17 @@ def save_tables(self, is_hiveserde: bool = False):
                 continue
             table_type = table.table_type.value if table.table_type else ""
             table_format = table.data_source_format.value if table.data_source_format else default_table_format
+            storage_location = table.storage_location
+            if table_type == "MANAGED":
+                storage_location = ""
```

Comment:
technically, even managed tables have a location

Reply:
i think the logic in fixture considers if type as managed then attaches a dbfs location instead of the one being passes. hence I had to overwrite it